internal/server/statvar/fetcher/formula_fetcher.go

```diff
@@ -39,83 +41,108 @@ func FetchFormulas(
 	remoteRespChan := make(chan *pbv2.NodeResponse, 1)
 	// Fetch for BT and SQL.
 	errGroup.Go(func() error {
-		data, _, err := v1pv.Fetch(
-			errCtx,
-			store,
-			[]string{"StatisticalCalculation"},
-			[]string{"typeOf"},
-			0,
-			"",
-			"in",
-		)
-		if err != nil {
-			return err
-		}
 		statCal := []string{}
-		for _, nodes := range data["StatisticalCalculation"]["typeOf"] {
-			for _, node := range nodes {
-				statCal = append(statCal, node.Dcid)
+		nextToken := ""
+		for ok := true; ok; ok = (nextToken != "") {
+			statCalReq := &pbv1.TriplesRequest{
+				Node:      "StatisticalCalculation",
+				Direction: "in",
+				NextToken: nextToken,
+			}
+			statCalResp, err := triples.Triples(
```

Comment:
Not for this PR but something to think about for the future.

The next token approach necessitates sequential behavior. Could we come up with a design that allows us to fetch multiple pages concurrently?

Reply:
Spoke offline: we decided that since this is just loaded once on Mixer start, it's okay to do the multiple requests